### DISCUSSION TITLE
[ARCHIVED] Collection Activation - Databases

### DISCUSSION THREAD

**Discussion by periode** [periode/augmenting-gallery#8] [Announcements]
Pick a museum website and look through their online database (you can start with the [museum databases page](https://github.com/periode/augmenting-gallery/wiki/Museum-databases)).

Post the one you would like to activate here, with a link and a paragraph of why you find it interesting, and what kind of app you would design around it.

**Comment by aidanmassie** [comment]
British Museum -> https://www.britishmuseum.org/collection

I chose the British Museum because of its extremely vast collection. As the website says, the collection is "two million years of history, across six continents." Because of its vastness, the museum offers a unique opportunity to share a lot of information about a lot of different artifacts throughout history. As an app, I think it would be interesting to have a database search, similar to what exists on the collection website already, that would yield results based on keywords. From there, rather than show everything on a long page, it would be helpful for users to sort similar artifacts into categories together. For example, whenever searching "Rome", there are immediately hundreds of alabastrons. By sorting into categories, the user could easily scroll through and find what specific kind of thing they're looking for. More specifically, when finding the artifact, it would be cool to show a visualization of where it came from on a map, where it's been (that they have a record of), and how it ended up in the museum. Also, an AR component of having the artifact being placed on a table/floor/equivalent plane would be interesting, allowing the user to walk around it and maybe even be able to zoom into it, spin it, etc. This would allow the online museum collection to be a little more interactive for users, while also making the objects more realistic for people all around the world without having to see them in person.

**Reply by periode** [reply]
Interesting idea of optimizing the search process! You should look into existing search paradigms (e.g. how do "related objects" work? what does machine learning have to in the searching process? what kinds of parameters from the object can be leveraged to make the search better?)

Secondly, regarding the AR part, I would actually investigate the relationship between spatial elements and search (how do representing things in 3D have to do with the searching/retrieval process? can we represent objects through lines/graphs/clutters/etc.).

And, in the wireframe itself, you can just focus on one specific use-case/search term

**Comment by yylinj** [comment]
China National Silk Museum: https://www.chinasilkmuseum.com/index_en.aspx

I find this museum interesting because I love the exhibition and enjoyed visiting the museum very much. It is also closely related to my everyday life since I love beautiful clothes and fabrics. It has an enormous collection of silk fabric and clothing ranging from 400 years before BC to contemporary. Because of the enormity of its collection, as the leading institute on textile, art history, and culture study, it has a diverse range of research topics. It has both online and real exhibitions with permanent collections and objects on lease from other museums. It is also one of the few Chinese museums with a pretty compelling and thought-through English version of the website. However, there is still some information missing regarding the exhibition object descriptions. I also noticed that in the exhibition, all the objects are behind glasses and hard to observe closely, especially when the light shines directly to the mirror and creates glaring reflections. I want to build an APP on one of the permanent exhibition, "The Way of Chinese Silk: Silk History and the Silk Road." It shows a detailed history of Chinese silk fabric and weaving techniques from around 400 BC to today. I want the App to focus on displaying the picture of the textiles and giving the audience a more direct view of the fabric and its design without the glass window of the museum. I hope I can find 3D objects of these fabrics to implement an AR aspect where the viewers scroll their phone and see different sides of the fabric and cloth, or zoom into extreme detail to see delicate patterns. Clothing is an everyday object. I want to re-create, through the App, a sense of personal belongings and intimacy with the fabric and the clothes. Especially because Chinese fabrics usually have symbolistic patterns (like for good fortune, long lives, or happiness) on them. So through the clothes and inspecting their patterns closely, hopefully, I can tell a story about them.

**Reply by periode** [reply]
This sounds good! You should definitely do a bit more research about the fabrication process and decoration process of chinese silk, so that you can focus on the interesting parts (does it start from the worm production? the dyeing process? the weaving?). Through this research process, it will help you figure out what story you'd like to tell.

And think about how you would represent the silk in the AR space: is it floating? worn by a human model? folded? a combination of all? Is it one close-up or a lot of them far far away? What would be the best use of displaying it in such a way?

**Comment by miafan716** [comment]
Suzhou Museum >> https://www.szmuseum.com/En/Home/Index 

Suzhou Museum has been famous on Chinese social media platform because of its modern design for a while. I visited it last summer and was amazed by its movement design and interaction parts. 
One of its strong point is that it has diverse exhibitions. The permanent exhibiton is about Suzhou and Zhejiang province history. The works include the archeologic discoveries. Normally, there are maps illustration and captions nearby. The special exhibtion focuses mainly on the current trend of multimedia. The one when I was visiting used beautiful cutout of papers to establish a mirror-reflecting 3d space and interpreted the Wu culture in a modern way. The immersive experience was really great. 
I would like to build an app to make the museum more lively. During my visit, I found the permanent exhibition and special exhibition were a little bit splited. They were like two shops in a shopping mall. You can find the way but can't really see the connection between. I would like to use the app to pop up pictures of some related work in other exhibition rooms to suggest the audience that there is a relationship between each exhibitions. Also, Suzhou Museum created a lot of physical objects and use sensors to trigger audios or videos so that the children could pay more attention and memorize the history better. With the app that I want to design, I really want to have a knowledge database to store every point that the children, or the user, want to look repeatedly even after their visit. The last point that I want to improve is that there are some objects displayed in the window with a mirror under them to show the details on the bottom. I am thinking whether an interactive AR projection of this project on the phone would be more convienient and easier to look through.

**Reply by periode** [reply]
It looks you have three ideas in there! Making a connection between the two exhibitions, making a knowledge database for children, and allowing for the 3D manipulation of objects—you should pick one!

Potentially, you could combine the two first ideas together: the knowledge graph is what makes the connection. But what examples of connection can you find? How do they get discovered by the audience? Is it like a constellation that one explores, or a puzzle that one builds up slowly through their visit? And how do you visually represent the connection between objects?

**Comment by chloeumlaut** [comment]
https://sis.modernamuseet.se/en/search/hilma%20af%20klint

In connection with our discussion on the intangible, I'd like to do my project on the Hilma af Klint room at Moderna Museet. This is the only permanent collection I could find of Klint's work, and it details several of her sketches as well as several paintings in her "Serie V." While I'm not an expert (I plan on becoming one) I know that Klint used ritualistic practices (including seances) to plan and produce her works. Her attention to spiritual guidance kept her work out of the public eye for many years, and as a result she was not exhibited until 40 years after her death. For this reason, she was not considered a pioneer of abstract, geometric, and symbolic art, though her work predates that of other "pioneers" of the style such as Kandinsky. After further research, I would like to build an AR app that implements aspects of the rituals behind her process (perhaps a sort of AR version of her setup? followed by sketches, then final paintings) that guides users through the ritual --> sketch --> painting --> creation of series process.

**Reply by periode** [reply]
That's a solid idea to start from. Once your research allows you to recreate one of those rituals, also think of how you include the audience: do they have to interact to let the prcess unfold, or is it happening automatically? do they end up painting on a canvas by touching/dragging the screen, or does one of the original paintings appear immediately?

**Comment by zanderzel** [comment]
I’d like to focus on the Galleria dell’Accademia di Firenze ( Accademia Gallery ) in Florence, Italy. It houses some of my personal favorite sculptures in the world and provides a fruitful environment to experience the mastery of Michelangelo Buonarroti. While the museum is full of works by many artists of the Rennaissance and other eras, I chose to focus on the Tribune, which is the area that houses The David, a 5.17 meter tall marble sculpture that represents the Renaissance era and the hall leading up to the Tribune, which houses Michelangelo’s unfinished sculptures of “slaves”. Being in an environment where you can see what is considered to be utmost perfection ( David ) and four uncompleted works, gives viewers an unprecedented view of the humanity that is involved. A lot of the time, when you just see a completed work, it’s easy to forget that somebody spent years of their life crafting a single piece. While I don’t think any reproduction of Michelangelo’s sculptures will ever capture the gravity as being in the same room as them, having an interactive AR app will get viewers much closer than a simple image or video. I’m imagining an app that has 3D scans of the four unfinished pieces, along with the David at the end of the hall - maintaining the original arrangement of the Gallery, with an add element of interactivity where users can pinch-to-zoom, “walk around” the sculpture in hopes to get a gauge of the sheer size.

Link to gallery: https://www.galleriaaccademiafirenze.it/en/art-archive/

**Reply by periode** [reply]
The concept of unfinished piece is an interesting one, but the proposal of having what is essentially a 3D model viewer is slightly limited in scope. For instance, you could think of an app where users could contribute their unfinished pieces, either directly through the app, or through social media hashtags, etc. You could also focus on the process, showing how pieces get finished over time, starting with a block of marble and ending up with David, for instance.

I'm assuming that your main focus will be on sculpture, since your working with the academy, so it might be worth researching a bit more the sculpting process of the time, to see what are the interesting things that you'd like to communicate to the audience.

**Comment by mdl601** [comment]
The Louvre isn't usually known for its contemporary art exhibits, but with the Richard Mille Art Prize, the Louvre Abu Dhabi was able to present a number of contemporary artists in its exhibition of "Memory, Time, Territory." The exhibition has 7 artists on display, all whose work has to do with time, the passage of time, and our relation to it as humans. I'm interested in going more in depth with the capabilities of taking a conceptual concept such as time, and bringing into the AR space in a way in which we can interact with it. The exhibit is also dynamic, and the structure of it could use more cohesiveness throughout thematically, in order to get to know the artists as individuals and how their pieces interact with the others as well. I will focus on Nasser Alzayani’s installation 'Watering the distant, deserting the near', which is made of compressed sand tablets produced via a laser-cut stencilling method. The sculpture is fragmented and old looking- but was made using some of the most innovative technology of our time. I'm not entirely sure how exactly I want to make this more interactive, but I do want users to be able to choose tablets to see them closer (instead of having to get alarmingly close to the piece itself) while also showing the perspective of creation that the artist would have gone through, combining elements of "old" and "new" and the passage of time through an interactive lens.

Gallery- https://www.louvreabudhabi.ae/en/Explore/art-here
Article- https://www.thenationalnews.com/arts-culture/art/2021/11/17/louvre-abu-dhabis-new-contemporary-art-show-ponders-memory-and-territory/
Artists page- http://www.nasseralzayani.com/2019_05_watering-the-distant-deserting-the-near_viii

**Reply by periode** [reply]
That sounds great, and it looks like your app is going to be on the border of the artistic, and not just the information (and that's totally fine). You could consider the app as itself an expression of memory and time, as the UI or UX can express the passing of time: can the buttons decay and disappear if one taps them too much? can some content only be viewable if the battery is below a certain level? What are some aspects of time in the UAE (golden hours, calls to prayer) or in a museum (opening times, closing times, suspension of time for the pieces)?

That being said, I would strongly recommend that you integrate _all the pieces_ in the exhibition, rather than just the one by Al Zayani.

**Comment by fractalessense** [comment]
CSMVS -- --> https://csmvs.in/

I would like to do a collection activation of the Chhatrapati Shivaji Maharaj Vastu Sangrahalaya (CSMVS) Museum of Mumbai, India. I was drawn to the Asian art and curious about where around the world the items have come from. I am imagining a map that traces the origins of objects. A slider on the screen would allow to explore visually the temporal scale: when items were created, how they traveled. This would allow for a coherent picture of the object’s geographical relations, and for a historical traceability of the object. The have on the website a well-organized catalogue, which I will use as the basis, and reformat it for a more georgaphically-interactive experience.

**Reply by periode** [reply]
It looks like you already a somewhat clear idea in your mind, and the two-dimensional serach axes is super interesting! If the relations between all the objects of the collection might be long and tedious to catalogue and organize, you could focus on one or two subsets of the collection (a bit like how the british museum does it [here](https://britishmuseum.withgoogle.com/)).

Following the itinerary of the object could also be a good idea, as the relations get revealed with other objects it encounters along the way.

**Comment by joshblecherman** [comment]
The gallery I chose is the museum of Islamic Art in the Pergamon Museum. This exhibit resonated with me due to the intricacy and geometry behind the art. I visited this museum last week and really enjoyed it. I imaging an AR app that displays the various art pieces that represent the geometric-heavy forms of Islamic art around the user using augmented reality. The user can then select an art piece and explore some of the interesting mathematics, and delve into the historical and mathematical context of the piece. Once the user selects a piece, the app will likely switch to 2D as the user explores the flat pattern of the work. The resources I will use for the images and 3d models for the project are the online exhibit for the museum of Islamic Art as well as any relevant 3d models I can find on SketchFab. For information, I will use websites that either discuss the mathematical context of the pieces or the historical context of the pieces. For more interactivity, maybe users can even build their own Islamic Art inspired patterns. 

[link to the online exhibit](https://universes.art/en/art-destinations/berlin/museum-of-islamic-art/)

**Reply by periode** [reply]
Yes, there are definitely a couple of code libraries which explore these intricate patterns: try to maybe find two or three to act as examples of the app, perhaps one about the [moucharabiya](https://i.pinimg.com/originals/16/25/dc/1625dc8357fe9319a196fa955acd6631.jpg), [dome volume](https://justfunfacts.com/wp-content/uploads/2016/05/mosque%E2%80%93cathedral-of-cordoba-dome.jpg) or the specific calligraphic script.

Also make it clear whether you're focusing on the Islamic art, on mathematics, or on the relationship between both (which then extends beyond simple architectural patterns).

**Comment by brianajpierre** [comment]
The museum I would like to work on is  The State Hermitage Museum, the website offers a thorough search bar criteria for when you are looking for a specific artwork. However, I think it would be cool if they offered a more efficient search to access specific information about the artworks you are looking for. The Hermitage offers a excellent search bar for you to put in vague concepts you want to see in an artwork. My design would take it further by having useful information about the representation used in the works, how that relates to the time period and the artist’s background. Furthermore, I would also include a comparison and contrast of other works within that time period. Hermitage already has sections that are clearly categorized, so what I am proposing would just improving its cohesiveness and improving its learning experience. 

An example of what I am proposing is, someone would search a painting that has flower meadow and knights. Then artworks fitting that description would appear. If the painting is already linked to a subcategory, the person would click that, and the information of social-economic influence ( materials used, who commissioned the artists, political influences), common visualization seen from this time period, and other works would appear. (Since this is already a function) I will include a subcategory of ‘Similarities’ and ‘Differences’. 
Link : https://www.hermitagemuseum.org/wps/portal/hermitage/explore/artworks?lng=en

**Reply by periode** [reply]
The Hermitage Museum is indeed a good choice in terms of searchability criteria. It will be important for your proposal to (1) identify which search criteria are proposed and why they're limited, then (2) propose different search criteria and justify why those would be important as well and, finally (3) how this connects back to augmented reality (i.e. how can spatiality play a role in the search process).

In your wireframe, you should at least include 2 or 3 search examples (the flower meadow and knight painting being only one of them).

**Comment by aashnasoni** [comment]
I chose the Victoria and Albert Museum, a leading museum in art and design, because they have a vast collection of accessories that would seem fun to activate. For example, the collection activation would give a deeper understanding of the form and history of their hats and shoes along with the fashion collection. It would be interesting to see how the website could be designed differently to specifically cater to these collections through the lens of evolution. Understanding the way fashion has evolved in its shape and form through AR would be really engaging as people could use technology to maybe see how they would look in the clothes and use AR as a lens to gain more knowledge. The fashion collection of the Victoria and Albert Museum spans five centuries and it would be cool to see how the accessories and clothes can be seen within different contexts and spaces. 

link: https://www.vam.ac.uk/collections/shoes

**Comment by izzyrieken** [comment]
Museum der Unerhörten Dinge - Isabelle Rieken

I chose this museum because it is simply exactly what I love - random items that tell random stories. I love the collection that seems to have to bias and inclusive of all random stories. There is something to be said about materialism as well and how all things/junk/materials can have meaning. 

The app: I would use the app to build context around the items. I am imagining myself as the audience at this museum and asking what I would enjoy seeing. The item is obviously the core of the experience, so I do not want to just recreate what’s in front of me, instead, I would like the perhaps show what the items look like before their wear-and-tear. That coupled with unique animations that artists can make to bring those items to life would be interesting to see.

**Comment by Dav1dyang** [comment]
[The Museum of Broken Relationships](https://brokenships.com/) - David Yang

I was trying to find the next destination for a little adventure on google maps. And just I was about to stop randomly looking around places I hadn't heard of before, I saw the museum of broken relationships in Zagreb, Croatia. The museum has a collection of stories of breakups, alongside an object that helps tell the story. This museum is an ongoing research project. People can contribute to the project by adding their stories to the website.

The app - I would like to focus on creating an audio narrative experience with AR for visual aids. The idea is that people can scan their 3D items or add a picture, record a voice memo or type out words to tell the story, and share it with others. Here is a quick Figma prototype of the app: https://www.figma.com/proto/b7DSq5GPU94Ikf07cZHbVI/AG---The-Museum-of-Broken-Relationships?node-id=2%3A44285&scaling=scale-down&page-id=0%3A1&starting-point-node-id=2%3A44285

**Comment by kylewenchang** [comment]
[V&A Wallpapers ](https://www.vam.ac.uk/collections/wallpaper) - Kyle Chang

I was slightly surprised when I saw the wallpaper section of the V&A Museum’s website, as I had never seen a wallpaper collection in an art museum before, and I hadn't really formally thought of them as art even though not much separates them from other art besides their presentation and maybe repetition. While reading through V&A’s description of its wallpaper collection, a section that struck me was: “Fragile, ephemeral, and easy to replace, wallpaper has often disappeared from the historical record.” There was an interesting tension for me between how the passing nature of wallpaper often causes it to be forgotten, and how museums have the ability to make us remember by giving us direct experience with an object in our space and in our time. I wanted to extend this by giving the user an opportunity to experience the works presented in their original context, as wall dressings and not framed in a gallery, by replacing the wall planes in the user’s camera view with walls decorated with the scanned wallpaper. By putting the work in its original context, you are also changing the world of the museum, which I feel allows the wallpapers to exist in a state that's more true to their purpose.

**Comment by kenparks12** [comment]
Kendall Parks -  [National Museum of African American History - Clothing](https://nmaahc.si.edu/explore/collection/search?edan_q=%2A%3A%2A&edan_fq%5B0%5D=topic%3A%22Clothing%20and%20dress%22&edan_local=1) 

I think it would be interesting to create an AR wardrobe for the pieces in this collection. People would be able to select items from the closet and get a bit of information about it, which ties in with the concept of putting things in its organic environment (something I love to play around with). 

This museum was quite spectacular when I went a few years ago; it was the first time that I saw African American culture being displayed and observed in a serious way, so it would also be a passion project if I got to make something using this database.

**Comment by olivestew** [comment]
Olive Stewart (ogs220) https://artsandculture.google.com/story/dgWBVjFuiU4wJA

I would like to explore the 'Puerto Rico Plural' exhibit through the Museo de Arte de Puerto Rico. I find this exhibit interesting because it explores the identity of the country through the lens of a wide group of different natives from different generations through different mediums. It explores identity from the 18th century until the current day and opens up many political and social conversations. There is a lot of disruption in identity in Puerto Rico due to colonialism, natural disaster, gentrification, and migration. However, this exhibit sees it as more of a plural existence rather than a fragmented one. Viewers can witness the damage the Hispanic-American war caused the country to face and the artistic protest that it inspired. I have many friends who are native Puerto Ricans and we often discuss the issues that they feel these effects caused the island and the pain of the US seeing their land as a commodity rather than a home. I feel like the art is visually stunning and would open many conversations. 

The application would allow viewers to click on different time periods, which would bring them into the art of the decade. Then they could learn about the current issues of the time through audiovisual effects and testimonials from people discussing these issues.

**Comment by aacarcrash** [comment]
Working with @hannah-y-zhao

Our collection was inspired by this line I found in the page for [Museum of Unheard Things](https://www.atlasobscura.com/places/museum-of-unheard-of-things)

"[BERLIN](https://www.atlasobscura.com/things-to-do/berlin-germany) SEEMS TO BE FASCINATED with the examination of “things.” The [Museum der Dinge,](http://www.atlasobscura.com/places/museum-der-dinge) a collection of everyday items devoted to modern object design, and the [Design Panoptikum](http://www.atlasobscura.com/places/design-panoptikum), a surreal museum of extraordinary industrial objects, are a couple examples. This beloved curio collection of “Unheard of Things” is further proof."


We wanted an exhibition of objects that aren't necessarily real objects but rather inspired by a concept like "unheard things". We came up with the idea for a museum displaying German fusion words like Handschuhe (hand shoe i.e. gloves) or Staubsauger (dust sucker i.e. a vacuum cleaner). As we are working in AR, we have the liberty to be creative with our interpretations of these objects as well. Hence, for example for EIsbar we might make a bear out ice, rather than displaying a polar bear.

**Comment by Opal42** [comment]
I want to build my app around works from the [Museum of Bad Art (MoBA)](http://museumofbadart.org/collections/). I found the premise of the collection really fascinating, with their tagline: 'art too bad to be ignored'. I think that building an app that explores the idea of critique, value judgements, and asking the question of "What is good/bad art?" would be really interesting. In my honest opinion, I think that many of the pieces on display at MoBA aren't _that bad_, and it is never quite clear whether the museum serves to display "bad art" as a rejection of traditional gatekeeping and exclusivity of museums, or rather as a means of public ridicule. 

My big struggle right now is figuring out how to do that in AR, in a way that wouldn't have been more productive in another form instead. I am considering doing something with spatial audio as a reflection of the critical voices in the art industry, but my experiences with spatial tracking in AR have been very poor thus far, so I worry it won't be viable.

**Comment by hannah-y-zhao** [comment]
Working with @AakSin 

We are interested in a collection of objects that aren't necessarily real or functional, but rather inspired by really long/fused German words that describe something specific. In addition to the examples that Aakarsh gave, our collection is also reminiscent of concepts of [Chindogu](https://pen-online.com/design/chindogu-the-art-of-subverting-useful-objects/) or [Carelman](https://www.itsnicethat.com/articles/impossible-objects).

Like Aakarsh said, we are interested in using the compound words but not with their intended meaning, such as Zahnfleisch (aka gum, aka tooth meat).

**Comment by Yupu-Chen** [comment]
Hong Kong Museum of Art >> https://hk.art.museum/en/web/ma/collections/ks-lo-collection-of-tea-ware-and-seals.html

I am interested in this collection about tea wares and seals from Hong Kong Museum of Art. In this collection, there are many tea pots and wares of various purposed and shapes. There are also a few personal seals. With this collection, I could explore the tea culture of China and Chinese seals as well as their significance regarding identities and Chinese cultures.  The shapes, colors, materials of these tea wares and seals all tell different stories.

App: About the app, I imagine that I could create an application that allows people to put these tea wares and seals on their own tables with AR functions. I would possibly make people leave their own comments  or even upload the pictures of their own tea wares or sales if they have one. What I am concerned about it the 3D objects that I could use for this app. I do not think that I could find the exact 3D models of the objects in the collection above. Therefore, there's a need to think about what objects I should use,

**Comment by daniaezz** [comment]
The museum I would like to build this app around is the [World Famous Crochet Museum.](https://www.atlasobscura.com/places/world-famous-crochet-museum) I am interested in this collection mainly because I enjoy crocheting, but also because I haven’t really thought about crochet as “art” before. 

I imagine that the AR app would help the visitor delve deeper into the objects on display, showing them the different types of stitches used and the colour palette or general theme of what the object is supposed to be. Is it a stuffed toy? Is it a needle holder? A clothing item? The app can also help illustrate how these things can be found in homes of people who crochet and what they bring to the place they are/used to be in.

**Comment by cz2109** [comment]
I have a broad framework for how I might want to execute this final project. I was inspired by the International Red Cross and Red Crescent Museum's (https://www.redcrossmuseum.ch/) use of AR, especially in their method of telling the stories of real people who have faced human rights injustices by having the visitor place their hand on the person's hand (on a screen) and triggering an audio recording. I wanted to do something similar where in an app, the viewer is able to trigger an audio recording while in direct, emotional confrontation with an object, person, etc.

The subject matter is still TBD, but as of right now, I am leaning towards creating a showcase/case study on a specific artist/art movement/art school (Subjects I have been toying around with are mirroring a collection off of Frida Kahlo, Antoni Gaudi, Ai Weiwei, Neo-Gothic/Art Nouveau movements, or The Bauhaus School, ...).

Another idea, that would be similar to the ICRC museum, is telling the stories of those who have faced diaspora in either Berlin/NYC, specifically, but not limited to East Asian experience, or another historical/social/political subject of similar gravity.

**Comment by emersonmack** [comment]
With this project, I want to explore storytelling within museum spaces. How does AR show a narrative that cannot be conveyed through a classic museum setting? I find inspiration in the [Mmuseumm's collection](https://www.mmuseumm.com/) in New York City which holds a wide variety of objects that rely on "story" to show importance. One of the notable exhibits I saw there was called "World-Leader Used Tissues". This display consisted of five crumpled up tissues in clear plastic boxes with world-leaders like Donald Trump and Queen Elizabeth labeled on them. This fabrication of artifacts still holds significance because of the stories and perspective attached to them. 
I would like to look at other museums and collections for this project but really like the idea of exploring storytelling strategies.
